ast.cpp

```diff
@@ -42,6 +58,7 @@ class IntConstExprNode : public ExprNode {
  public:
   IntConstExprNode(int val) : val_{val} {}
 
+  void CodeGen() {}
```

Comment:
```suggestion
  void CodeGen() override {}
```
Make the signature more explicit and can catch typo bugs, such as `void Codegen()`.